### PR TITLE
Add scheduled workflows for test suites

### PR DESCRIPTION
**Pull request by Polber**
This PR will set up various automated test workflow for tracking test health through the use of status badges on the main repo page. Tests will be scheduled to run every 6 hours and test reports will be attached to each run similar to Java PRs.

### CODE REVIEW

**Review by damccorm — COMMENTED** [review]
Are these just identical tests to the existing suites? Can we just set up schedule triggers on the existing suites so that we're not duplicating code?

File: .github/workflows/java-pr.yml [hunk 1]
```diff
@@ -27,8 +27,6 @@ on:
       # This will make it easier to verify action changes don't break anything.
       - '.github/actions/setup-env/*'
       - '.github/workflows/java-pr.yml'
-  schedule:
```
Comment (Abacn) [inline .github/workflows/java-pr.yml:30]: same as comments of #1475, generally we would like to have same workflow run for different triggers. Any reason this didn't work?

**Comment by manitgupta** [comment]
Are we not going this anymore?

**Comment by manitgupta** [comment]
Nevermind I saw the new PR

**Pull request closed without merging**